sw/device/silicon_creator/rom/e2e/BUILD

```
@@ -687,3 +689,75 @@ test_suite(
         slot,
     ) for t in BOOT_POLICY_BAD_MANIFEST_CASES for slot in SLOTS],
 )
+
+[genrule(
+    name = "empty_test_slot_{}_corrupted_{}_bin_signed_{}".format(slot, device, key),
```

Nit.
I think it easier to read this way:
```suggestion
    name = f"empty_test_slot_{slot}_corrupted_{device}_bin_signed_{key}",
```

---

unfortunately bazel does not support f-strings.